ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -133,6 +134,19 @@ type opennebula_ceph_datastore = {
     "rbd_format"                ? long(1..2)
 };
 
+@{ 
```

Comment:
all the annotations must be fixed according to http://quattor-pan.readthedocs.org/en/latest/pan-book/pan-book.html#annotations. in particular, the annotation for a `type` has to start with `@documentation{`, should be simpel search and replace